content/history.md

```diff
@@ -4,6 +4,17 @@ layout: single
 title: History
 ---
 
+## [1.8.1] - 2024-02-26
+
+### Fixed
+
+- Update the minimum required version of `packaging` ([#9031](https://github.com/python-poetry/poetry/pull/9031)).
+- Handle unexpected responses from servers that do not support HTTP range requests with negative offsets more robust ([#9030](https://github.com/python-poetry/poetry/pull/9030)).
+
```

Comment:
We should maybe add a `{{ note }}` about disabling lazy wheels if any issues are encountered.

Reply:
I'm not convinced that would help much but feel free to add a note if you like. 🙂